### DISCUSSION TITLE
这到底是chatgpt几啊，他回答我是2？

### DISCUSSION THREAD

**Discussion by 1205129045x** [GaiZhenbiao/ChuanhuChatGPT#420] [Q&A] [answered]
我选的模型是`gpt-3.5-turbo`，但是问它是chatgpt几，他回答我是**2？**
![image](https://user-images.githubusercontent.com/50540240/227975642-f529a3cf-dcb1-438c-8e9a-429500123f07.png)
是不是用api调用就是这个吊样

**Comment by GaiZhenbiao** [comment, marked as answer]
看回答就知道这不是2能生成的质量……不用纠结这种问题，用的就是3.5 :)
